contracts/LiquidityToken.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.24;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC20Permit, Nonces } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import { ERC20Votes } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+contract LiquidityToken is ERC20Permit, ERC20Votes {
```

Comment:
I don't see any harm in making it burnable as well

Reply:
the only reason I hesitate is because of the "meme" of having 75M total supply, burnable would mean we could end up with not a nice round number. Maybe a question for Bryan?

Reply:
good point. Let's not make it burnable